exchanges/kraken/kraken_websocket.go

```diff
@@ -1024,11 +1126,66 @@ func (k *Kraken) wsAddOrder(request *WsAddOrderRequest) (string, error) {
 	return resp.TransactionID, nil
 }
 
+// wsCancelOrders cancels one or more open orders passed in orderIDs param
 func (k *Kraken) wsCancelOrders(orderIDs []string) error {
```

Comment:
When inserting in incorrect orderID: It works as expected but there are some transient issues that are occurring.

```
gocryptotrader\cmd\gctcli> go run ./... cancelbatchorders --exchange=kraken --orders_id="OFOC35-3I7FB-DJPLBX,thisisbad,O2NK2T-OF5DQ-4QOXCE" --pair="btc-usdt" --asset=spot
2020/11/17 13:12:34 rpc error: code = Unknown desc = Kraken, has been cancelled 2 orders of 3:
exit status 1
```

```
[ERROR]|WEBSOCKET| 17/11/2020 13:12:34 |routines.go exchange Kraken websocket error - Kraken Websocket status for RequestID 189367398: 'EOrder:Invalid order' // I don't think we need to log this error
[ERROR]|WEBSOCKET| 17/11/2020 13:12:34 |exchange not found 
[ERROR]|WEBSOCKET| 17/11/2020 13:12:34 |exchange not found
```

Reply:
Dear shazbert,  about "On startup there are some similar websocket transient issues with log output when loading in new orders."  - I drew attention to this earlier, this problem is present in the master, I think it does not apply to my PR

Reply:
Thanks for clarifying! Will add a TODO. 👍